docker/advanced_settings.md

```diff
@@ -335,111 +295,43 @@ The predict endpoint will return a prediction response in JSON. It will look som
+* **max_workers** - 
+* **job_queue_size** - Number of requests that can be queued. This queue is shared across models.
+* **number_of_gpu** - Number of GPUs available for model server when serving inferences on GPU hosts
+* **keystore** - SSL Key Store
+* **keystore_pass** - SSL password
+* **keystore_type** - Store of cryptographic keys and certificates
+* **private_key_file** - Location of the private key file
+* **certificate_file** - Location of the certificate file
 
-```text
-    [MMS arguments]
-    --models
-    squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
-
-    --service
-    optional
-
-    --gen-api
-    optional
-
-    --log-file
-    optional
-
-    --log-rotation-time
-    optional
-
-    --log-level
-    optional
-
-    [Gunicorn arguments]
-    --bind
-    unix:/tmp/mms_app.sock
-
-    --workers
-    optional
-
-    ##Following option is used only for GPU and is present in mms_app_gpu.conf
-    --num-gpu	     
-     optional
-
-    --worker-class
-    gevent
-
-    --limit-request-line
-    0
-
-    [Nginx configurations]
-    server {
-        listen       8080;
-
-        location / {
-            proxy_pass http://unix:/tmp/mms_app.sock;
-        }
-    }
-
-    [MXNet environment variables]
-    OMP_NUM_THREADS=4
-```
-
-## Configuring SSL
-`THIS SECTION IS EXPERIMENTAL`
-
-To safely send traffic between the server and the client, you need to setup a secure sockets layer for nginx server.
-
-First of all, you need to get a SSL certificate. It includes a server certificate and a private key. Suppose you have generated the cert (or received one from a CA) and resulting two files are located at /etc/nginx/ssl/nginx.crt and /etc/nginx/ssl/nginx.key.
-
-Second step is to create a Docker container which exposes TCP port 443 for SSL:
-
-```bash
-docker run -itd --name mms -v /home/user/models/:/models -p 8080:443 -p 8081:80 mms_image:latest
-```
-
-Note that you expose both https and normal http ports.
-
-Third step is to modify nginx section of `mms_app.conf` file to add ssl settings:
-
-```text
-server {
-    listen       80;
-    listen       443 ssl;
-
-    ssl_certificate /etc/nginx/ssl/nginx.crt;
-    ssl_certificate_key /etc/nginx/ssl/nginx.key;
-
-    location / {
-        proxy_pass http://unix:/tmp/mms_app.sock;
-    }
-}
-```
-
-Now you can try both https://your_public_host_name:8080/ping or http://your_public_host_name:8081/ping to test the service.
-
-```bash
-curl -X GET https://your_public_host_name/ping
-```
-
-or
+in the range of 0 .. (num-gpu-1) in a round-robin fashion. **By default MMS uses all the available GPUs but this parameter can be configured if user want to use only few of them**. A discussion on how to set this parameter can be found [here](../docs/optimised_config.md)
 
-```bash
-curl -X GET http://your_public_host_name/ping
-```
+```yaml
+vmargs=-Xmx1g -XX:MaxDirectMemorySize=512m -Dlog4j.configuration=/opt/ml/conf/log4j.properties
```

Comment:
Most of properties should be commented out.

Reply:
We don't want to misleading user even in example.
keystore related parameter won't be used for http
and it doesn't make sense both key file and keystore ad defined